src/snowflake/connector/compute_chunk_size.py

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+from __future__ import annotations
+
+import logging
+import math
+
+logger = logging.getLogger(__name__)
+
+constants = {
+    "CURRENT_CHUNK_SIZE": 8388608,
+    "MAX_OBJECT_SIZE": 5497558138880,
+    "MAX_PART_SIZE": 5368709120,
+    "MIN_PART_SIZE": 5242880,
+    "MAX_PARTS": 10000,
+}
```

Comment:
they can be flatten out, no need to be put into a dictionary.

Reply:
Done